.travis.yml

```diff
@@ -6,3 +6,6 @@ notifications:
     on_success: change  # options: [always|never|change] default: always
     on_failure: always  # options: [always|never|change] default: always
     on_start: false     # default: false
+rvm:
+ - "2.1.2"
```

Comment:
No need for the `rvm` config. We can use Travis' [support for .ruby-version](http://docs.travis-ci.com/user/languages/ruby/#Using-.ruby-version).